autoload/airline/themes/nord.vim

```diff
@@ -78,9 +78,15 @@ let g:airline#themes#nord#palette.visual = airline#themes#generate_color_map(s:V
 let g:airline#themes#nord#palette.visual.airline_warning = s:VWarn
 let g:airline#themes#nord#palette.visual.airline_error = s:VError
 
-let s:IAMain = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
-let s:IARight = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
-let s:IAMiddle = [s:nord5_gui, s:nord1_gui, s:nord5_term, s:nord1_term]
+if g:nord_uniform_status_lines == 0
+  let s:IAMain = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
+  let s:IARight = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
+  let s:IAMiddle = [s:nord5_gui, s:nord1_gui, s:nord5_term, s:nord1_term]
+else
+  let s:IAMain = [s:nord5_gui, s:nord1_gui, s:nord5_term, s:nord1_term]
```

Comment:
I get your idea of adding some visual effect to make clear there are multiple segments, but when the _uniform status line_ setting is enabled the background color of the `IAMain` and `IARight` elements should stay at `nord3`. It smoothly integrates with the `IAMiddle` element while the active status line is still easily recognizable due to the highlighting with colors from the [Frost][] palette.

<p align="center">Enabled <em>uniform status lines</em></p>
<p align="center"><img src="https://user-images.githubusercontent.com/7836623/61274528-46781880-a7ac-11e9-9353-89d06bbb68e7.png" /></p>

<p align="center">Disabled <em>uniform status lines</em></p>
<img src="https://user-images.githubusercontent.com/7836623/61274527-45df8200-a7ac-11e9-8290-b16595cc332c.png" /></p>

```suggestion
  let s:IAMain = [s:nord5_gui, s:nord3_gui, s:nord5_term, s:nord3_term]
```

[frost]: https://www.nordtheme.com/docs/colors-and-palettes#frost

Reply:
I glad you said it! 😃I was not sure myself.